open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaClient.java

```diff
@@ -9,65 +9,191 @@
 
 import java.util.List;
 
+/**
+ * @param <T> Glossary artifact type for supplied and return.
```

Comment:
Please could you add a description for this class. 

I suggest something like. This is the Subject Area client API, for use by the subject area expert. This API exposes SCRUD (search, create, read ,update, delete) operations that can be performed on a Glossary Artifact. A Glossary Artifact is a {@link Node} or a {@link Line}.

Reply:
okay

Reply:
@davidradl  let's omit "A Glossary Artifact is a {@link Node} or a {@link Line}.". And leave only "I suggest something like. This is the Subject Area client API, for use by the subject area expert. This API exposes SCRUD (search, create, read ,update, delete) operations that can be performed on a Glossary Artifact."

Reply:
Because anytime can to appear a new Glossary Artifact which is not node or line.

Reply:
@immmus I don't think we will as the idea is that the nodes and lines are displayed on a graph. We may have more types of lines and nodes. I like this line as it is definitive definition of Glossary Artifact and has hard links to the Node and Line definitions - so there is no ambiguity. I suggest we can add this in a subsequent issue.